backend/models/game.go

```diff
@@ -1,12 +1,9 @@
 package models
 
-import "time"
-
 type Game struct {
-	Id           uint64
 	Name         string
 	Author       string
-	CreationDate time.Time
+	CreationDate string
```

Comment:
Long term, I think it would be better if this field is a Time field and not a string, but perhaps it would be better to keep it a string until a PR is made to convert the value in the seed data.

Reply:
I'll just manually convert it to string when we store it, then convert it back to time when we load it from the db.

- [ ] Convert Time object at db level

Reply:
Mongo and Mongo tests are now working with the time.Time creationDate. We'll still have to test Firestore later.